composer.json

```diff
@@ -13,10 +13,10 @@
     "require": {
         "php": ">=5.3.2",
         "ext-curl": "*",
-        "composer-plugin-api": "~1.0.0-alpha10"
+        "composer-plugin-api": "^1.0"
```

Comment:
This plugin is not compatible to composer < 1.0.0-alpha10. 
`^1.0` means `1.0.0 <= version < 2.0.0`.
I want `1.0.0-alpha10 <= version < 2.0.0`.